### ISSUE TITLE
annotateVariants will be marked as complete if the pipeline crashes before Variant Calling

### ISSUE THREAD

**Issue by Seth-Karlo** [CuppenResearch/IAP#68]
The check here:
https://github.com/CuppenResearch/IAP/blob/master/illumina_annotateVariants.pm#L132

creates:
```
if [ \"\$(tail -n 1 $preAnnotateVCF | cut -f 1,2)\" = \"\$(tail -n 1 $outvcf | cut -f 1,2)\" ]
```
Which will return true if both of these files are empty or do not exist at that point in time. As the pipeline does not stop if an earlier tool crashes, this will create the done file and then skip annotation on subsequent runs. Wrapping this in an if [ -s  $preAnnotateVCF ] style check would fix this issue.

**Comment by rernst** [comment]
Good catch, thanks!

**Comment by rernst** [comment]
Fixed: accb97032acc5c3a3a42cf27df5c024c68fc6db6
